test/run_all_tests.py

```diff
@@ -119,20 +119,6 @@ def main():
                 'expected_image': '',
                 'expected_output_files': 1,
             },
-        'export_moved_cube' :
-            {
-                'script'        : os.path.join(current_path, 'export_cube.py'),
-                'test_scene'    : os.path.join(current_path, 'test_scenes/cube_moved.blend'),
-                'expected_image': os.path.join(current_path, 'expected_results/cube_moved.png'),
-                'expected_output_files': 6,
-            },
-        'export_rotatedX_cube' :
-            {
-                'script'        : os.path.join(current_path, 'export_cube.py'),
-                'test_scene'    : os.path.join(current_path, 'test_scenes/cube_rotatedX.blend'),
-                'expected_image': os.path.join(current_path, 'expected_results/cube_rotatedX.png'),
-                'expected_output_files': 6,
-            }
```

Comment:
why remove these tests?

Reply:
These are to be replaced by more specific tests. For instance, tests for rotation are comprised of rotations on different axis, by different angles and in different rotation ordering. You can check this in #6